deployment/environment/crib/ccip_deployer.go

```diff
@@ -339,8 +432,50 @@ func setupChains(lggr logger.Logger, e *deployment.Environment, homeChainSel uin
 		),
 	)
 	if err != nil {
-		return *e, fmt.Errorf("failed to apply changesets: %w", err)
+		return *e, fmt.Errorf("failed to apply EVM changesets: %w", err)
 	}
+
+	if len(env.SolChains) > 0 {
```

Comment:
```suggestion
        if len(env.SolChains) > 0 {
        deployedEnv := testhelpers.DeployedEnv{ ... }
        buildConfig := ccipChangesetSolana.BuildSolanaConfig{ ... }
	for solCS, _ := range env.SolChains {}
```


This does the same thing for skipping if there are no solChains, but also allows you to iterate for more solchains than just the [0] index